InsiteTeamTask.Services/Configuration/ServicesMappingProfile.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoMapper;
+using InsiteTeamTask.Data.DTOs;
+
+namespace InsiteTeamTask.Logic.Configuration
+{
+    public class ServicesMappingProfile : Profile
+    {
+        public ServicesMappingProfile()
+        {
+            CreateMap<MemberDto, AttendenceDto>();
```

Comment:
Here you're mapping a `MemberDto` to an `AttendanceDto`, which I don't think is going to work